src/components/SelectionCart/SelectionCart.js

```diff
@@ -52,13 +55,32 @@ class SelectionCart extends Component {
             'reservation': [
             ]
         };
+        let reservedComponents = {
+            'memberID': this.userID,
+            'activeComponents': [
+            ]
+        }
         for(let id in addedItems) {
```

Comment:
Add spaces between declarations for for, if and else if statements
```suggestion
        for (let id in addedItems) {
```